app/src/modules/content/routes/item.vue

```diff
@@ -302,7 +302,7 @@ export default defineComponent({
 			return hasEdits.value;
 		});
 
-		const { confirmLeave, leaveTo } = useEditsGuard(isSavable);
+		const { confirmLeave, leaveTo } = useEditsGuard(hasEdits);
```

Comment:
@nickrum please explain me 🥲
Why do we need theses lines, do you know?
```js
			if (isNew.value === true) {
				return Object.keys(defaults.value).length > 0 || hasEdits.value;
			}
```

Reply:
> Also, can't we just rely on hasEdits or isSavable, why do we need both? 

It's for the navigation guard. We only want to show the _Are you sure you want to leave?_ when you've made edits. However, an item can be Savable without having Edits 👍🏻

Reply:
Ohhh makes sense now, because it may be automatically populated or a clone of other item.
Thank you for the explanation ❤️